resources/migrations/000_migrations.yaml

```diff
@@ -8648,6 +8657,846 @@ databaseChangeLog:
                   constraints:
                     nullable: false
                   defaultValue: false
+  - changeSet:
+      id: 321
+      author: jeff303
+      comment: Added 0.42.0 - modify type of activity.details from text to ${text.type}
+        on mysql,mariadb
```

Comment:
is this supposed to be part of the comment? Maybe you should make it all one line or use one of the multiline markdown directives just to be safe

Reply:
This is valid YAML (the "plain style" as described [here](https://stackoverflow.com/a/21699210)).  And this was even generated by our YAML library, using code from here: `https://www.notion.so/metabase/App-DB-Migrations-Tips-and-Tricks-b29b4a8048964ac0a0685b8e6ff19fa0`

But I can see about bulk updating the style to something else instead.